chromium_src/third_party/ethash/src/lib/keccak/keccak.c

```diff
@@ -7,4 +7,4 @@
 #undef __x86_64__
 #endif
 
```

Comment:
should be ```chromium_src/```**brave**```/third_party/ethash/src/lib/keccak/keccak.c```

and there is a typo in a comment above: "needd on macOS"

Reply:
FixedFixed in 444c3346445a8615688dab4532d41889d99c648e